package.json

```diff
@@ -71,6 +82,7 @@
     "@babel/core": "^7.23.5",
     "@babel/plugin-transform-modules-commonjs": "^7.23.3",
     "@babel/preset-typescript": "^7.23.3",
+    "@figma/code-connect": "^1.0.0",
```

Comment:
`@figma/code-connect` is needed as a dependency from root because we are using a singular `.env` file for both react and react native using the `--config ` flag.